test/functional/shada/marks_spec.lua

```diff
@@ -248,4 +249,26 @@ describe('ShaDa support code', function()
     eq('', funcs.system(argv))
     eq(0, exc_exec('rshada'))
   end)
+
+  it("updates deleted marks", function()
```

Comment:
Use single quotes, so
```suggestion
  it('updates deleted marks', function()
```